core/lib/spree/core/engine.rb

```diff
@@ -44,6 +45,11 @@ class Engine < ::Rails::Engine
         Migrations.new(config, engine_name).check
       end
 
+      # This code can be injected by an external gem/part of Solidus:
```

Comment:
I don't get the comment, can you please expand a little bit?

Reply:
I meant that this code (the initializer callback) could be moved to another gem, for example to a solidus extension/part that is responsible of sending emails. 

More in general adding subscriptions using an initializer seems to me reasonably simple and convenient, it may be the standard way to do this... what do you think?

Reply:
Ok, I agree that it can be moved but I'd remove the comment then.

Reply:
👍